src/commands/change-java-version.yml

```diff
@@ -0,0 +1,30 @@
+description: |
+  Change default java version from OpenJDK v11 to v8.
+parameters:
+  java-version:
+    type: integer
+    default: 8
+    description: |
+      The version of OpenJDK to change to
+steps:
+  - run:
+      command: |
+        JAVA_VER=$( java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1 )
```

Comment:
You almost always want to wrap your subshell commands in double-quotes.  There is a very succinct explanation with examples [here](https://www.oreilly.com/library/view/linux-shell-scripting/9781785881985/f3497351-d8d0-4de2-be20-4f169b59ebe3.xhtml).